tests/unit_tests/test_min.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def mean_process_code():
```

Comment:
PR looks good, there just seem to be some leftovers in the method naming, e.g. `mean_process_code`, `test_mean`. 

Same for `test_sum`.

In `test_multiply` the function is called `divide_process_code`. 

FFTR